dist/js/orb.js

```diff
@@ -355,6 +355,11 @@ require('./queries');
                         if (column.type === 'Reference') {
                             self.references[column.name] = undefined;
                         }
+
+                        // update the base attribute
+                        if (!self.isNew() && _.has(options, column.field)) {
+                            self.baseAttributes[column.name] = options[column.name];
```

Comment:
this _should_ solve the problem of loading defaults into the model, validated in latest release